service.datadog.yaml

```diff
@@ -1,22 +1,185 @@
 ---
-schema-version: v2
+schema-version: v2.1
 dd-service: datadog-agent-nightly
+contacts:
+  - type: slack
+    contact: https://dd.slack.com/archives/C06PQ7H6PHP
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Add the channel as a comment so we know where this is pointed